### ISSUE TITLE
[BUG] WCAG Accessibility issue in AI Foundary's PromptFlow Chat - text contrast

### ISSUE THREAD

**Issue by DrGriff** [microsoft/promptflow#3876]
**Describe the bug**
The colour scheme isn't easy to read, it's not.

**How To Reproduce the bug**
Steps to reproduce the behavior, how frequent can you experience the bug:
1. Run a chat with the LIGHT theme

**Expected behavior**
I would expect this to meet WCAG standards, but it's almost unreadable with black text on dark blue

**Screenshots**
![Image](https://github.com/user-attachments/assets/e2e29650-3051-47eb-8dc1-5c0cda0f0996)

**Running Information(please complete the following information):**
Don't think this is relevant
 - Promptflow Package Version using `pf -v`: [e.g. 0.0.102309906]
 - Operating System: [e.g. Ubuntu 20.04, Windows 11]
 - Python Version using `python --version`: [e.g. python==3.10.12]

**Additional context**
How did this pass any sort of basic QA testing before release to web?

**Comment by DrGriff** [comment]
what extra information you need?

**Comment by DrGriff** [comment]
Rather than have an automated response say "there's not been any activity", why doesn't someone on this project actually READ this and action it.
